test/paws_mock.js

```diff
@@ -117,6 +118,16 @@ const HEALTCHECK_SUBSCRIPTION_FILTERS = {
         }
     ]
 };
+const MOCK_LOGS = [
+    {
+        "CreationTime": new Date().toISOString(), "Id": "c5d8e7ea-90b0-4549-9746-f67c8f6c00", "Operation": "UserLoginFailed", "Workload": "AzureActiveDirectory",
+        "ErrorNumber": "500121", "LogonError": "AuthenticationFailedSasError"
+    },
+    {
+        "CreationTime": new Date().toISOString(), "Id": "c5d8e7ea-90b0-4549-9746-f67c8f6c11", "Operation": "UserLoginFailed", "Workload": "AzureActiveDirectory",
+        "ErrorNumber": "500101", "LogonError": "AuthenticationFailedSasError"
+    }
```

Comment:
cosmetic: please format these in the usual JSON style with one property on each line 

```javascript
{
    CreationTime: new Date().toISOString(),
    Id: "c5d8e7ea-90b0-4549-9746-f67c8f6c00",
    // etc
}
```